examples/tasks/advanced/images/image_LICENSE.txt

```diff
@@ -3,7 +3,7 @@ licensed accordingly to the original license:
 
 Project                 License                   File
 ----------------------------------------------------------------------------
-Enthought               BSD 3-Clause              LICENSE.txt
+Enthought               Proprietary
```

Comment:
I think we should not put proprietary files in open source projects. Not sure though, @rkern any feedback ? The Enthought name and the logos are trademarked so people cannot use them indiscriminately. What kind of license do these files need? Btw, we should also mention somewhere that the Oxygen icons are LGPL licensed